conference.js

```diff
@@ -687,47 +668,55 @@ export default {
             });
     },
 
-    startConference(con, tracks) {
-        tracks.forEach(track => {
-            if ((track.isAudioTrack() && this.isLocalAudioMuted())
-                || (track.isVideoTrack() && this.isLocalVideoMuted())) {
-                const mediaType = track.getType();
+    async startConference(conP, tracksP, connectionAndTracksPromiseFunction) {
```

Comment:
Can you please add some JSDocs here ? Its hard to follow the logic and not very clear that this function either takes the connection and tracks objects or a promise that returns those two.

Reply:
Done!